docker/Dockerfile.fedora-base

```diff
@@ -9,11 +9,15 @@ RUN dnf install -y \
     automake \
     cpio \
     elfutils-devel \
+    elfutils-libs \
```

Comment:
This shouldn't be needed because it will be pulled in by elfutils-devel as one of its dependencies.

Reply:
Also removed.